### ISSUE TITLE
Review how to map outcome of shorter CURIE for CRS in CIS

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/coverage-implementation-schema#18]
Awaiting results from NA discussions:

https://github.com/opengeospatial/NamingAuthority/issues/92
https://github.com/opengeospatial/NamingAuthority/issues/93

The `srsName` would need to be updated or a new property added to allow for CURIEs, potentially a list of CRSes to be compounded, or even a definition in PROJJSON or the upcoming JSON encoding of CRS according to WKT2.

**Comment by pebau** [comment]
Fortunately, the coverage standard has to be crafted agnostic of CRS description mechanisms. srsName expects an identifier string as per CIS 1.1. So there is no issue here, nothing to do.

**Comment by jerstlouis** [comment]
@pebau See discussion [here](https://github.com/opengeospatial/NamingAuthority/pull/119#issuecomment-885599677) and [here](https://github.com/opengeospatial/NamingAuthority/pull/119#issuecomment-885618131) as to what would need to be done.

> So, where un-registered compound CRSs built from registered CRSs need to be supported, I would allow an array of CRSs, so the value in a query parameter would be [epsg:4979],[ISO-8601:Gregorian_UTC].

> Well, it would have to be a change in CIS anyhow since the value range would have to be extended from "URI" to "URI or OGC CURIE" or - in the original proposal - to "URI or some OGC-specific-string".

> Yes, srsName could be extended for such on-the-fly compound CRS and CURIEs. That would require a GML revision, if done properly.

**Comment by pebau** [comment]
hm, this is (i) specific for a particular language (JSON) and alien to users coming from other languages. Besides, it is not what current tools support already. Maybe we should go where the users are.

**Comment by jerstlouis** [comment]
@pebau Well no it is not specific to JSON -- other object definitions have arrays concepts, and in XML you would simply repeat the tag multiple times with the different values.

**Comment by pebau** [comment]
yes, exactly that's my point too - we want to express an ordered sequence aka array, question is jsut the syntax. ArcGIS and GDAL AFAIK both use "epsg:4326+mysweetcrs:42", therefore this has been suggested in ticket #93.

**Comment by jerstlouis** [comment]
@pebau but as mentioned in that discussion I referenced above, the proper way to do this in JSON would be `"crs" : [ "epsg:4326", "mysweetcrs:42" ]` and in XML it would be something like `<crs>epsg:4326</crs><crs>mysweetcrs:42</crs>`. This would support the CURIE approach, where each of these array elements can either be a CURIE or a full URI, not sticking a `+` buried into a string that requires additional parsing rather than using the built-in ordered sequence construct of the encoding language.

**Comment by pebau** [comment]
actually, that demonstrates it: 3 different syntaxes we now have on the table. I have my personal preference as you sense, but that does not matter - rather, I'd hope that OGC-NA comes up with a decision on #93 that carries across features, coverages, etc.

**Comment by pebau** [comment]
FYI, in CIS 1.1 the CRS type (srsName) is string, so open to all ideas 19111 has and might have in future :)
